content/docs/rfcs/35/README.md

```diff
@@ -49,10 +49,10 @@ The fields that are concatenated as part of the `payload` Waku2 field are:
  - `protocol-id`: identifies the protocol in use (1 byte). 
  Supported values are:
      - `0`: none or omitted;
```

Comment:
Can we add the version for ChaChaPoly symmetric case only? So applications don't need to revert to previous version for this use case